lbrynet/daemon/DaemonServer.py

```diff
@@ -54,10 +54,9 @@ def start(self, use_auth):
 
     @defer.inlineCallbacks
     def stop(self):
-        if self._api is not None:
-            yield self._api._shutdown()
-        if self.server_port is not None:
```

Comment:
This happens in [Daemon._stop_server()](https://github.com/lbryio/lbry/blob/b464853ede24cb03043b921c58dbef1e4d9a3192/lbrynet/daemon/Daemon.py#L351) which is called in [Daemon._shutdown()](https://github.com/lbryio/lbry/blob/b464853ede24cb03043b921c58dbef1e4d9a3192/lbrynet/daemon/Daemon.py#L406)